charts/ingress-nginx/values.yaml

```diff
@@ -700,6 +700,7 @@ controller:
       annotations: {}
       # prometheus.io/scrape: "true"
       # prometheus.io/port: "10254"
+      labels: {}
```

Comment:
Can you please document what is this for? :) 

Take a look into the other examples, and specify what is this field and why it would be used on a helm chart.

Thanks